resources/demos/FabCar/FabCar.obs

```diff
@@ -0,0 +1,146 @@
+contract Car {
+  string color;
+  string make;
+  string model;
+  string owner;
+
+  Car@Owned(string color, string make, string model, string owner) {
+    this.color = color;
+    this.make = make;
+    this.model = model;
+    this.owner = owner;
+  }
+
+  transaction getColor() returns string { return color; }
+  transaction getMake() returns string { return make; }
+  transaction getModel() returns string { return model; }
+  transaction getOwner() returns string { return owner; }
+
+  transaction toString() returns string {
+    return "{color='" + color + "',make='" + make + "',model='" + model + "'',owner='" + owner + "'}";
+  }
+}
+
+// I couldn't find anything about maps or arrays
+// in the documentation, but we can make linked lists!
+contract CarList {
```

Comment:
Can you use extend the polymorphic linked list in the standard library to do what you want? It doesn't support search, but you can adapt functionality from ERC20/Dict.obs to do that. In particular:
* move ERC20/Dict.obs to Obsidian_Runtime/src/main/java/Runtime/edu/cmu/cs/obsidian/stdlib
* Adapt the technique used there to make a LinkedList of comparable items
* put that new linked list in the standard library

Reply:
Can we go ahead and merge this, and you can do the remaining work in a separate PR?